### DISCUSSION TITLE
Supplemental MRF Information

### DISCUSSION THREAD

**Discussion by jayfeins** [CMSgov/price-transparency-guide#355] [Q&A] [answered]
Upon initial launch of this entire MRF initiative, there was discussion we could include “footnotes, disclaimers, data dictionaries and other explanatory language” within files. How would we get this information on our MRF's? For example, we have clients that would like to include special  messaging regarding Anesthesiologists.

**Comment by shaselton-usds** [comment, marked as answer]
There is now an attribute `additional_information` in the [Negotiated Price Object](https://github.com/CMSgov/price-transparency-guide/pull/446) that can be leveraged to handled what you've described here.
